### DISCUSSION TITLE
Правила/тест для полиции

### DISCUSSION THREAD

**Discussion by octothorpteam** [octothorp-team/public-dobrograd#543] [Предложения]
#### Приветствую

* * *

Все мы знаем о насущной проблеме связанной с полицией.
 Начну с простого примера. Возьмем обычного игрока Доброграда которое наиграл уже некоторое время для разблокировки профессий полицейского, он бежит в полицейский департамент к заветной доске и берет проффесию и снаряжение, зачитывает пару строчек правил, а что дальше? Дальше ничего, игрока пускают в свободное плавание, в следствие которого он от банального не знание своей проффесий, плохо отыгрывает свою роль, мешает своим коллегам и гражданам так же администраций которым приходится обьяснять  банальные термины по типу “Маркировка” “Тен-коды” и т.д.
 Я предлгаю ввести на уровне octohelp’а небольшой гайдлайн который введет игрока в основы отыгровки своей профессий, в этом гайдлайне будут устоявшийся термины которые часто используют игроки доброграда  в отыгровке полиций. Так же хочу дополнить что я предлагаю именно **гайдлайн** а не правило за которое сможет наказывать администрация, думаю можно как-то еще подумать как сделать данный “гайдлайн” более назойливым для прочтение.
 UPD: По поводу форума, если смотреть даже со стороны бывалого игрока не на видном месте лежать подобные гайдлайны.
 Если тот же бармен будет отыгрывать неправильно это в малейшей степени навредить чем в том же случае с полицией.

- Надо
- Не надо

0голосов

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/wezd/32/240_2.png) [WezD](https://forum.octothorp.team/u/WezD) ([#1285](https://forum.octothorp.team/p/1285))


Неплохая задумка. На вики можно сделать статью для новичков ,а после ознакомление пройти небольшой тест на знание своей профессии. ![:cloud_with_lightning:](https://forum.octothorp.team/images/emoji/twitter/cloud_with_lightning.png?v=10 ":cloud_with_lightning:")

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/warsucksletsparty/32/2109_2.png) [warsucksletsparty](https://forum.octothorp.team/u/warsucksletsparty) ([#1375](https://forum.octothorp.team/p/1375))


постараюсь до конца лета написать с [@Sighty](/u/sighty) гайдлайн для офицеров полиции, о котором нас давно просил Инквизи и Доктор.

может, это дело конечно тоже неплохое, но все же, приложим с Сайти свои старания и напишем, как играть, что делать и т.д.

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/bespa/32/289_2.png) [bespa](https://forum.octothorp.team/u/bespa) ([#1441](https://forum.octothorp.team/p/1441))


Можно попробовать сделать тест как при первом заходе на сервер и если ты его не прошёл тебе отказывают в работе полицейского департамента на определённое время.

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/bob/32/285_2.png) [bob](https://forum.octothorp.team/u/bob) ([#1604](https://forum.octothorp.team/p/1604))


Классная идея.


> тебе отказывают в работе полицейского департамента на определённое время.


Можно ввести кулдаун на час.

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/bespa/32/289_2.png) [bespa](https://forum.octothorp.team/u/bespa) ([#1844](https://forum.octothorp.team/p/1844))


![](https://forum.octothorp.team/user_avatar/forum.octothorp.team/bob/40/285_2.png) bob:


> Можно ввести кулдаун на час.

Хорошая идея.

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/chelog/32/9_2.png) [chelog](https://forum.octothorp.team/u/chelog) ([#2063](https://forum.octothorp.team/p/2063))


Я очень надеюсь, что когда мы введем систему организаций, лидеры полиции сами смогут определять, как принимать и обучать сотрудников. Ну а пока что можно обойтись каким-то гайдлайном, как и написал Варсак, если нужен будет какой-то небольшой механ, мы добавим, как только информация будет готова:

![](/user_avatar/forum.octothorp.team/warsucksletsparty/40/255_2.png)[Новые игроки и полиция](https://forum.octothorp.team/t/novye-igroki-i-policziya/466/3) [Предложения](/c/dobrograd/suggest/8)


> постараюсь до конца лета написать с [@Sighty](/u/sighty) гайдлайн для офицеров полиции, о котором нас давно просил Инквизи и Доктор. 
> может, это дело конечно тоже неплохое, но все же, приложим с Сайти свои старания и напишем, как играть, что делать и т.д.

**Comment by travka4000** [comment]
Думаю лучше тебе просто давали курс и после этого тест ты сдавал, ведь тест когда ты ничего не знаешь? Я имею ввиду вот как академия дпд и тд
